### PR TITLE
Initial implementation using Qt5

### PR DESCRIPTION
**Pull request by felipesanches**
There's a lot yet to be done here.
WARNING: This commit includes absolute paths to resource files. We will
have to change that later.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+app = QApplication(sys.argv)
+
+content = QLabel("# Fontbakery-desktop\nv0.0.1")
+
+web = QWebEngineView()
+channel = QWebChannel(web.page())
+web.page().setWebChannel(channel)
+channel.registerObject("content", content)
+web.load(QUrl('file:///Users/fsanches/devel/github_felipesanches/fontbakery-desktop/resources/index.html'))
```
Comment (felipesanches) [inline main.py]: I am fully aware of this hardcoded path which I will need to change asap

File: main.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      fb_output = subprocess.check_output(fb_cmd, stderr=subprocess.STDOUT)
+
+    except OSError:
+      content.setText("ERROR: fontbakery is not properly installed!")
+      return
+    except subprocess.CalledProcessError:
+      pass
+
+    content.setText(open(md_file, "r").read())
+    # delete md_file!
```
Comment (felipesanches) [inline main.py]: also need to delete the temp file

File: main.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+run_fontbakery("/Users/fsanches/devel/github_google/fonts/ofl/lobstertwo/LobsterTwo-Regular.ttf")
```
Comment (felipesanches) [inline main.py]: and need to trigger this call from a file drop event instead of this hardcoded call in here

**Comment by eliheuer** [comment]
Thanks! I'll see if I can have any luck improving the UI/UX and making a DMG.

**Pull request merged**